src/inflators/model.tsx

```diff
@@ -139,6 +141,20 @@ export function inflateModel(world: HubsWorld, rootEid: number, { model }: Model
     }
   });
 
+  // TODO: Not sure if here is the best place and
+  //       this is the best way to inflate animations.
+  //       We may need to revisit.
+  if (model.animations !== undefined && model.animations.length > 0) {
+    inflateMixerAnimatable(world, rootEid, {
+      animations: model.animations
+    });
+    inflateLoopAnimation(world, rootEid, {
```

Comment:
This still makes sense to do here to support the behavior we have now of: "If a gltf has animations and does not have any loop-animation component, loop the first animation"